changelog/std-traits-hasStaticMember.dd

```diff
@@ -0,0 +1,13 @@
+Added `std.traits.hasStaticMember` to check whether a symbol is a static member of a type.
```

Comment:
I don't see any deciding arguments one way or another.

I think enforcing that the second line is blank would be good to avoid accidental ambiguity, and would follow formatting for git commit messages.